.github/workflows/codeql.yml

```diff
@@ -0,0 +1,75 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: 'CodeQL'
+
+on:
+  push:
+    branches: ['main', archive/*, main*, release-*, release/*]
```

Comment:
We might not want to run this on release-* and instead only run it on release-1.13.* and up instead, because those have automation in place that we control.

Reply:
That's fair, I've pushed an update that should match 1.13.* and higher